tests/pems/crl/test_s2nc.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+# Ensure that s2nc accepts all generated certificates
+
```

Comment:
Should generate.sh call this? Or is it just here for manual testing?

Reply:
It was just for manual testing, but I guess it makes sense for it to call it. I added it: f591286a86f0cfbcdd829c5f62e4938b5e9d5270